lang-python/legacy-cgi/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=legacy-cgi
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="python-build hatchling python-installer wheel"
+PKGDES="Fork of the standard library cgi and cgitb modules, being deprecated in PEP-594"
+
+NOPYTHON2=1
+ABSPLITDBG=0
```

Comment:
This is an architecture-agnostic package, replace this line with:

```
ABHOST=noarch
```

To indicate as such.